README.md

```diff
@@ -1,29 +1,29 @@
 
 - __Geo-Replication__  
-TiKV uses Raft to support Geo-Replication. We have ported [etcd's raft] (https://github.com/coreos/etcd/tree/master/raft) implementation to Rust. By using raft, Key-Value regions always stay consistent. Placement driver (or pd) is also important for Geo-Replication. Pd collects state information from heartbeats of all Raft groups' leaders regularly. According to load feedback and isolation judgment, pd can rebalance data automatically and smoothly. So Geo-Replication can be achieved, and makes TiKV always stable and balanced across regions. 
+TiKV uses Raft and placement driver to support Geo-Replication. 
 
 - __Horizontal scalability__  
-By carefully designing Raft groups and placement driver, we enable horizontal scalability in TiKV. A region, similar to spanner’s directory, is the base unit of data movement in TiKV. A region is controlled by one Raft group and its size is limited to 64 MBs . To achieve horizontal scalability, a region can split into two regions when the data size exceeds the limit automatically. Placement driver keeps the metadata of all regions. To ensure horizontal scalability, we ensure the metadata of each region is only 100s of bytes. Together with other techniques, TiKV can easily scale up to hold 100s of TBs of data.
+With carefully designed Raft groups and placement driver, TiKV excels in horizontal scalability and can easily scale up to hold 100s of TBs of data.
 
 - __Consistent distributed transactions__  
-Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions. Placement driver plugs in a set of TS system, therefore timestamp oracle will be allocated in global. When TiKV client send transaction request to TiKV server, server will check relavant meta from the pd's etcd, and then TS generate two valid timestamps in two separated handling phase implying start timestamps and commit timestamps. Before transaction ends, the start timestamp will be carrying on util the transaction is committed. Once transaction is committed, externally-consistent transactions succeed.
+Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions. 
 
 - __Coprocessor support__  
-TiKV will be accessiable and any application can develop on TiKV. We tend to provide a simple deployment solution soon after.
+You can develop any application on TiKV.
```

Comment:
How about "Similar to Hbase, TiKV implements coprocessor framework to support distributed computing."?